app/models/game.rb

```diff
@@ -7,5 +7,19 @@ class Game < ActiveRecord::Base
   
   has_many :pieces
   has_many :players
+
+  def piece_present?(x, y)
```

Comment:
Not a blocker, but typically you want to name your methods using:
* a short helping or linking verb (is, can, has);
* followed by a noun/pronoun (piece, she, his);
* optionally followed by a verb (present, eat, die);

From there it gets a bit murky, but usually you don't have to go past the three. Also, it is implicit that a method also describes the class in which it is in (is_piece_present[_in_game], get_piece[_in_game])